modules/virt-viewing-network-state-of-node-console.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * networking/k8s_nmstate/k8s-observing-node-network-state.adoc
+
+:_content-type: PROCEDURE
+[id="virt-viewing-network-state-of-node-console_{context}"]
+= Viewing the network state of a node from the web console
+
+As an administrator, you can use the {product-title} web console to observe `NodeNetworkState` resources and network interfaces, and access network details.
+
+.Procedure
+. Navigate to *Networking* → *NodeNetworkState*.
++
+In the *NodeNetworkState* page, you can view the list of `NodeNetworkState` resources, and the interfaces that are created on the nodes.
```

Comment:
```suggestion
In the *NodeNetworkState* page, you can view the list of `NodeNetworkState` resources and the interfaces that are created on the nodes.
```

Reply:
This sentence means `the list of NodeNetworkState resources and the corresponding interfaces'. 
corresponding = the interfaces on the nodes (hence that comma).

Any suggestion to make it sound better?

Reply:
that is a tricky one. Honestly, I think your use of "corresponding" in explaining it would clarify, i.e.:
```suggestion
In the *NodeNetworkState* page, you can view the list of `NodeNetworkState` resources and the corresponding interfaces that are created on the nodes.
```
What do you think?